pack.go

```diff
@@ -26,53 +26,111 @@ import (
 	"github.com/opencontainers/go-digest"
 	specs "github.com/opencontainers/image-spec/specs-go"
 	ocispec "github.com/opencontainers/image-spec/specs-go/v1"
-	artifactspec "github.com/oras-project/artifacts-spec/specs-go/v1"
 	"oras.land/oras-go/v2/content"
 	"oras.land/oras-go/v2/errdef"
 )
 
-// MediaTypeUnknownConfig is the default mediaType used when no
-// config media type is specified.
-const MediaTypeUnknownConfig = "application/vnd.unknown.config.v1+json"
+const (
+	// MediaTypeUnknownConfig is the default mediaType used when no
+	// config media type is specified.
+	MediaTypeUnknownConfig = "application/vnd.unknown.config.v1+json"
+	// MediaTypeUnknownArtifact is the default artifactType used when no
+	// artifact type is specified.
+	MediaTypeUnknownArtifact = "application/vnd.unknown.artifact.v1"
```

Comment:
This variable is for artifact type. Should it be named _**ArtifactType**UnknownArtifact_ instead of _**MediaType**UnknownArtifact_?

Reply:
Good point! My considerations are:
1. `artifactType` is actually a media type
> **`artifactType`** _string_
This property SHOULD be used and contain the mediaType of the referenced artifact. 
Reference: https://github.com/opencontainers/image-spec/blob/v1.1.0-rc2/artifact.md#artifact-manifest

2.  When `PackImageManifest` is true, `artifactType` will be used as the config media type
(See https://github.com/Wwwsylvia/oras-go/blob/be6588adc63a903be64ebb9ae6d5d06a1b7aad21/pack.go#L145)
> **`artifactType`** _string_
This OPTIONAL property contains the type of an artifact when the descriptor points to an artifact. This is the value of artifactType when the descriptor references an [artifact manifest](https://github.com/opencontainers/image-spec/blob/v1.1.0-rc2/artifact.md). This is the value of the config descriptor mediaType when the descriptor references an [image manifest](https://github.com/opencontainers/image-spec/blob/v1.1.0-rc2/manifest.md).
Reference: https://github.com/opencontainers/image-spec/blob/v1.1.0-rc2/descriptor.md